File: exercism/haskell/pangram/src/Pangram.hs
module Pangram
  ( isPangram
  ) where

import Data.Char (toLower)
import Data.Set (fromList, isSubsetOf)

isPangram :: String -> Bool
isPangram text = isSubsetOf alphabet (lowerSet text)
 where
  lowerSet = fromList . toLower'
  toLower' = map toLower
  alphabet = fromList "abcdefghijklmnopqrstuvwxyz"

File: exercism/kotlin/pangram/src/main/kotlin/Pangram.kt
class Pangram {
    companion object {
        val alphabet = "abcdefghijklmnopqrstuvwxyz".split("")

        fun isPangram(s: String): Boolean {
            return s.split("")
                    .map { it.toLowerCase() }
                    .toSet()
                    .containsAll(alphabet)
        }
    }
}
